dev/PushNotifications/PushNotifications.idl

```diff
@@ -118,22 +83,22 @@ namespace Microsoft.Windows.PushNotifications
         // Gets a Default instance of a PushNotificationManager
         static PushNotificationManager Default{ get; };
 
-        // Register an activator using an ActivationInfo context and cache the flag for unregister
-        void RegisterActivator(PushNotificationActivationInfo details);
+        // Registers an application for Notifications
+        // For Unpackaged applications, registers support for protocol activations
+        // For packaged applications with Background Infrastructure support, registers Push Trigger with the Background Infrastructure
+        // For packaged applications without Background Infrastructure support, registers a COM Service
+        void Register();
 
-        // Unregister any activator given PushNotificationRegistrationActivators
-        // 1) If kind = PushTrigger is specified, the trigger itself will be removed
-        // 2) If kind = ComActivator is specified, the Project Reunion Background Task component will no longer act as an InProc COM Server
-        // 3) If kind = ProtocolActivator is specified, the application will be unregistered from the long running process that handles activation
-        void UnregisterActivator(PushNotificationRegistrationActivators activators);
+        // Unregisters the COM Service
```

Comment:
// Packaged Apps: Force COM to launch a new process
// Unpackaged apps: Remove the Foreground Sink to force Long Running Process Singleton to launch a new process instance.